create_derivatives/helpers.py

```diff
@@ -0,0 +1,7 @@
+from pathlib import Path
+
+
+def check_dir_exists(dir):
```

Comment:
Outside of the scope of this app, but I'm wondering if this should be added to asterism?

Reply:
Possibly - it's potentially not even necessary to factor this out as a distinct method. Really all we're doing here is providing a custom exception message, which could also be done by catching the original exception (which I think would be an OSError) and re-raising.